website/docs/guides/microsoft-fabric-qs.md

```diff
@@ -54,8 +54,8 @@ A public preview of Microsoft Fabric in dbt Cloud is now available!
     CREATE TABLE dbo.customers
     (
         [ID] [int],
-        [FIRST_NAME] [varchar] (8000),
-        [LAST_NAME] [varchar] (8000)
+        \[FIRST_NAME] [varchar](8000),
```

Comment:
the line `[FIRST_NAME] [varchar] (8000),` in the code block doesn't render properly: 

![Screenshot 2023-11-29 at 10 03 23 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/d2ff7b43-c55a-47df-91ed-702911c03380)

to fix this, i needed to escape the `[` character for all instances in the code block

Reply:
Looks good now!